bin/yaml/cpp.yaml

```diff
@@ -804,6 +804,18 @@ compilers:
             chmod +x installer.bin
             ./installer.bin --unattendedmodeui none --mode unattended --prefix .
             rm installer.bin
+    intel-os-cpp:
+      type: script
+      dir: "intel-os-cpp-{name}"
+      check_exe: bin/clang++ --version
+      check_stderr_on_stdout: true
+      targets:
+        - name: 2023.0618
+          fetch:
+            - https://github.com/intel/llvm/releases/download/sycl-nightly%2F20230618/dpcpp-compiler.tar.gz dpcpp-compiler.tar.gz
+          script: &intel-os-install-script |
```

Comment:
This script appears to do nothing the normal installation process does...I _think_ this can be rephrased in terms of a normal install, which would fix the "daily" part too! Will sketch out soemthing!